okio/src/main/java/okio/Okio.java

```diff
@@ -242,4 +242,8 @@ static boolean isAndroidGetsocknameError(AssertionError e) {
     return e.getCause() != null && e.getMessage() != null
         && e.getMessage().contains("getsockname failed");
   }
+
+  public static Source tee(final Source source, final Sink copySink) {
```

Comment:
Let's please test resulting `Source`, since `TeeSource` is internal we can cover `Okio.tee()` instead of `TeeSource`!